apps/studio/src/components/PageEditor/constants.ts

```diff
@@ -19,15 +19,91 @@ export const DEFAULT_BLOCKS: Record<
       },
     ],
   },
-  accordion: undefined,
-  button: undefined,
-  callout: undefined,
+  accordion: {
```

Comment:
the purpose of this defintion here is to provide default values?

Reply:
It seems like most of the defaults are empty though. Would a better way to be to see the key e.g. `text` or `title` or `url` and then populate it?

Reply:
i think zhong jun had a PR fixing this. will remove on my end. we shouldn't do thsi approach (introspection of the object and then setting based on key) because these defaults are set broadly and are also very far removed from definition. 

i think the correct approach here would be to ask the schema itself for defaults (which it is able to do), so that each schema comes w/ sane defaults.

regardless, no impact here cos removing ._.